common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -117,6 +117,13 @@
     },
     "sqlState" : "42604"
   },
+  "AVRO_CANNOT_WRITE_NULL_FIELD" : {
+    "message" : [
+      "The record to be written to Avro contains null value for non-null field <name> with schema <schema> from Avro schema.",
```

Comment:
Cannot write null value for field "<fieldName>", which is defined as non-null (<schema>) in the Avro schema. 
To allow null values for this field, provide an Avro schema that includes "null" (e.g., a union type) 
using the 'avroSchema' option.

Reply:
Shall we use the Avro schema(json string) here? Or let's simply not show the schema.